Tests/ApiEndpoints/test_allhousingapp_pytest.py

```diff
@@ -90,27 +90,35 @@ def test_application_deleted(self):
         self.url = control.hostURL + 'api/housing/apartment/applications'
         self.data = {
             'ApplicationID' : -1,
-            'EditorUsername' : control.leader_username,
+            'EditorProfile' : {
+                'AD_Username' : control.leader_username,
```

Comment:
You may want to add Gender, though I don't remember if the service actually uses that information for anything, so it might be fine without it

Reply:
Thanks. The tests passed without it and a search for "Gender" in the HousingService turned up nothing, so I think it is fine.